internal/services/v1/permissions.go

```diff
@@ -915,14 +908,25 @@ func (a *loadBulkAdapter) Next(_ context.Context) (*core.RelationTuple, error) {
 		return nil, nil
 	}
 
-	a.current.Caveat = &a.caveat
-	a.current.Integrity = nil
-	tuple.CopyRelationshipToRelationTuple(a.currentBatch[a.numSent], &a.current)
+	if a.caveat.CaveatName != "" {
```

Comment:
I understand this is _correct_ but I wonder if this can cause some unexpected behavior to clients from the former loosely-defined contract.

Reply:
Caveats had to be specified before for context